go.mod

```diff
@@ -18,7 +19,7 @@ require (
 	github.com/test-network-function/l2discovery-exports v0.0.1
 	github.com/test-network-function/l2discovery-lib v0.0.5
 	github.com/test-network-function/privileged-daemonset v0.0.5
-	gopkg.in/yaml.v2 v2.4.0
```

Comment:
go mod has  both version , Did you run go mod update and go mod vendor?

Reply:
I ran go mod tidy and go mod vendor. I checked that yaml.v2 is coming from some other internal libraries. Not from my yaml parser code. Therefore, both stays.